processor/src/test/java/org/mapstruct/ap/test/ignorebydefaultsource/Source.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.ignorebydefaultsource;
+
+class Source {
```

Comment:
Interesting that having package protected works with our test harness. Good to know this 😄

Reply:
AFAIK Junit 5 tests are even recommended to be package private! Then you don't have 1000 different `Source` and `Target` :-) (But this actually only happened by accident :-D)

Reply:
Yes we do have 1000 different ones currently 😄 . I know about the tests, and that's what we are trying to do now. 

I wanted to make the mappers as well, but that doesn't work nicely with our test harness, we have some special class loading to make all this work correctly.